index.js

```diff
@@ -882,6 +885,7 @@ class Peer extends stream.Duplex {
     if (this.destroyed) return
 
     if (this._pc.signalingState === 'stable' && !this._firstStable) {
+      this._firstStable = false
```

Comment:
This boolean seemed to be set in the wrong place. Before it was being set after any signalingStateChange instead of after the first stable state. I moved it inside the if statement so it only fires after first stable.